### PR TITLE
Crusader Kite Shield

### PR DESCRIPTION
**Pull request by Xantholne**
Adds a Crusader Kite Shield which is craftable by hitting a buckler in hand with a pen to instantly start a crusade! Deus Vult!

Updated sprites:
updatedkite

### CODE REVIEW

**Comment by FlattestGuitar** [comment]
I'm sorry, but I really think we should be going the other way with the chaplain. 

He's already encouraged to hunt vampires in maint enough, he doesn't need the shield. If anything, he should be losing the armor and sword.

**Comment by taukausanake** [comment]
I was also thinking he doesn't need a new shield. Looks alright, but it's not needed especially if it's just a subtype with nothing new to it

**Comment by SheepFighter** [comment]
The chaplain is the equivalent of a parish priest, not a Knight Templar. He's already too militant as it is. 
That said, I see no problem with this being made an exclusive for traitorous men of the cloth.

**Comment by Xantholne** [comment]
Well. The shield is literally a buckler in stats. It's just to go with the Templar Armor and Holy Claymore to mkae it even more enticing for Chaplain players.

As it stands, it does nothing game breaky or making them more power gamey as a buckler is made from wood which they have much of just around them in the chapel even.

**Comment by FlattestGuitar** [comment]
It still encourages a certain playstyle by the Chaplain. 

It's comparable to people grabbing the Roman shield from the autodrobe. It's possible, but I'd rather not see it happen. It's rather concerning that players do this. 

We do have quite a few problems with the general powergamey approach of a lot of people and the RP pros of adding the shield will be thwarted by the powergame of the lower end of our RP spectrum. 

I don't see this as a positive addition.

**Comment by TullyBurnalot** [comment]
I second what Flattest has said. The Chaplain is a spiritual counselor, not a holy warrior. They do not need weapons and armour of war added to their inventory.

**Comment by Fox-McCloud** [comment]
main qualms are about the sprite itself:

- the shield looks like its floating on the mob's back
- From behind it looks like a rectangular object
- It's a bit grainy/low fidelity

Considering chaplains can already build a buckler at round start, I don't have an issue with this, personally.

File: code/game/objects/items/weapons/shields.dm [hunk 1]
```diff
@@ -49,6 +49,12 @@
 	burn_state = FLAMMABLE
 	block_chance = 30
 
+/obj/item/weapon/shield/riot/buckler/kite
+    name = "Crusader Kite"
+    desc = "WE WILL TAKE JERUSALEM."
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/shields.dm]: Deus Vult.

**Comment by FalseIncarnate** [comment]
I think this should (if anything) be something anyone can make by using crayons on a buckler or something, rather than a "unique" item the chaplain gets at roundstart.

If you go this route, you can even then add alternate designs/heraldry for shields based on the color used

**Comment by Regen1** [comment]
Agree with false, instead of adding it as a roundstart item, encouraging players to use it and giving everything to the player, we can make it so that you have to craft it.

**Comment by Doukhan** [comment]
This should be something that requires a little effort, ideal for a craft item but not another straight up buff (just combine buckler + bible, maybe a few crayons).
Also templar armour really needs nerfing, should be cardboard or at least hazard vest tier not semi-riot gear, I'm a fan of the church militant myself but it's so over and badly done now I'd much rather it just go away.

**Comment by Draconic-saint** [comment]
Yup, make it craftable and available for everyone. 

Give the people more things to play with without pushing the chaplain even more into the "DEUS VULT" role.

**Comment by Crazylemon64** [comment]
I think the opinions here that a means of making a buckler into this new "crusader shield", instead of the shield coming default with the chaplain, are a good idea - as otherwise, I feel this might legitimize the crusader aspect of the chaplain a bit too much.

**Comment by Fox-McCloud** [comment]
If this is to be changed, make it something you can change by using a pen on the buckler and it gives a list of transformations you can select or something of that nature.

File: code/game/objects/items/weapons/shields.dm [hunk 2]
```diff
@@ -48,6 +48,24 @@
 	materials = list()
 	burn_state = FLAMMABLE
 	block_chance = 30
+	var/redesigned = 0
+
+/obj/item/weapon/shield/riot/buckler/kite
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/shields.dm]: From here.

File: code/game/objects/items/weapons/shields.dm [hunk 3]
```diff
@@ -48,6 +48,24 @@
...
+
+/obj/item/weapon/shield/riot/buckler/attackby(obj/item/I, mob/user, params)
+    if(istype(I, /obj/item/weapon/pen) && !redesigned)
+        var/obj/item/weapon/shield/riot/buckler/kite/kite = new
+        to_chat(user, "<span class='notice'>You redesign [src] into [kite]!</span>")
+        user.drop_item()
+        user.put_in_active_hand(kite)
+        qdel(src)
+        return
+    ..()
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/shields.dm]: To here.

Are all spaces instead of tabs. Not allowed.

**Comment by Fox-McCloud** [comment]
Merge conflictsssss.

**Comment by Fox-McCloud** [comment]
Closing on request of submitter; may be resubmitted in the future.

**Pull request closed without merging**